client/src/main/java/org/apache/celeborn/client/read/RssInputStream.java

```diff
@@ -268,7 +269,11 @@ private boolean isExcluded(PartitionLocation location) {
     }
 
     private boolean isCriticalCause(Exception e) {
-      boolean isConnectTimeout =
+      boolean connectException =
```

Comment:
I don't know if there exist other kind exceptions, seems we can wrapper a Celeborn exception for network connect error.

```java
    ChannelFuture cf = bootstrap.connect(address);
    if (!cf.await(conf.connectTimeoutMs())) {
      throw new IOException(
          String.format("Connecting to %s timed out (%s ms)", address, conf.connectTimeoutMs()));
    } else if (cf.cause() != null) {
      throw new IOException(String.format("Failed to connect to %s", address), cf.cause());
    }
```

Reply:
Updated, please take a look again.